bp_common/test/src/random/mkrandfile.c

```diff
@@ -0,0 +1,116 @@
+#include <stdio.h>
```

Comment:
Probably would be easier to work with if it were just a Python script. But the main thing is that the file should get generated as part of the libperch build.

Reply:
I think the .txt would get installed in a folder BP_FILES=files/ at the sdk top level.  Maybe a few different sizes?

Reply:
> We'll want this as part of our https://github.com/black-parrot-sdk/perch

Not sure why we'll want to put this into perch/. I think 'mkrandfile' is more like a tool (ex: 'dramfs_mklfs') rather than something that can be integrated into a C library.

Reply:
How about this arrangement:

black-parrot-sdk / bp-demos: put the random test 'read_rand_test.c'(for Dromajo) here.
black-parrot-sdk / random: the original RNG system
black-parrot-sdk / random-files: install different sizes of random files here (BP_FILES=random-files/)